scania-digital-design-system/tegel
# feat(stepper): prefix change + singular capital case

**Describe pull-request**  
prefix change + singular capital case for stepper component

**Solving issue**  
Fixes: DTS-1648

**How to test**  
1. Go to the Storybook link below
2. Check that the component design is the same as in the main branch
3. Prefix is changed from sdds to tds 
4. in doc files we mentioned "Stepper" instead of stepper

**Suggested test steps**
